server/core/src/main/java/dev/slimevr/config/YawCorrectionConfig.kt

```diff
@@ -0,0 +1,9 @@
+package dev.slimevr.config
+
+class YawCorrectionConfig {
+	private var _amountInDegPerSec = 0.0f
```

Comment:
Could also add a toggle to allow simply toggling the option on/off without having to set it to 0. This would also allow the _amountInDegPerSec to have a sensible default value.

Also get/set are automatically generated in Kotlin and you don't need a private member + public propriety (see how other config classes were done)

Reply:
When JSON deserializes a missing field into the float (e.g. when loading a config from a previous version of the server), it sets it to `NaN`. `@JsonInclude(Include.NON_NULL)` did not seem to work.

Do you know how to fix this?

Reply:
@Erimelowo could you review https://github.com/SlimeVR/SolarXR-Protocol/pull/155 when you have a chance? Thanks!

Reply:
Addressed by adding enabled setting.